pkg/auth/rights/require.go

```diff
@@ -155,3 +155,62 @@ func RequireUser(ctx context.Context, id ttnpb.UserIdentifiers, required ...ttnp
 	}
 	return nil
 }
+
+// RequireAny checks that context contains any rights for the
+// given entity identifiers.
```

Comment:
Let's clarify that it requires "any rights for each of the given entity identifiers"